src/openforms/submissions/models/submission_value_variable.py

```diff
@@ -377,17 +383,25 @@ class Meta:
     def __str__(self):
         return _("Submission value variable {key}").format(key=self.key)
 
-    def to_python(self) -> Any:
+    def to_python(
+        self, value: JSONValue | date | time | datetime = empty
+    ) -> JSONValue | date | time | datetime:
```

Comment:
I *think* we also have `dateutil.relativedelta` (from the `rdelta` json logic operation) and possibly `timedelta` objects here in the types but I'm not sure.

Maybe it's a good idea to define a type alias for this?

```py
type VariableValue = JSONValue | date | time | datetime | relativedelta
```

I couldn't see an actual `timedelta` being returned by our json logic py, it's formatted to an ISO-8601 string

Reply:
Oh and this `empty` default value will not pass type checking I'm afraid, so it'll have to be added to the union:

```py
value: VariableValue | object
```

I don't like sentinels with type checking :(